teps/0096-pipelines-v1.md

```diff
@@ -332,6 +332,19 @@ Each V1 CRD should be fully documented before being released, but release of a V
 unrelated documentation issues.
 Once we've achieved the goals of this TEP, we can release a v1 software version of Pipelines.
 
+### Migration plan
+The v1 CRD is expected to be released and served in v0.42.0 while v1 would be the stored version in v0.43.0.
```

Comment:
specifying which releases we'll do this in is a bit too granular for the tep imo. You can just say we will start serving v1 versions, and then one release later, we will swap the stored version to v1, mark v1beta1 as deprecated, and document that the EOL for v1beta1 is 9 months after that release.